.eslintrc

```diff
@@ -319,13 +319,29 @@
         ]
       },
       "rules": {
-        "no-duplicate-imports": "off",
         "@typescript-eslint/no-duplicate-imports": "error",
+        "@typescript-eslint/no-shadow": "error",
+        "jsdoc/check-param-names": "off",
```

Comment:
Hm, will these be off for all the packages?

Reply:
Just for TypeScript files